tests/hotplug/affinity.go

```diff
@@ -42,7 +42,7 @@ var _ = Describe("[sig-compute]VM Affinity", decorators.SigCompute, decorators.S
 	})
 
 	Context("Updating VMs node affinity", func() {
-		patchVMNodeSelector := func(newNodeSelectorMap map[string]string, op string, vmName string, vmNamespace string) {
+		patchVMNodeSelector := func(newNodeSelectorMap map[string]string, op patch.PatchOp, vmName string, vmNamespace string) {
```

Comment:
Ahh... here the type is used as input which requires it to be public. Not nice but reasonable usage.

Reply:
No, not nice. I tried to reduce the public type as much as posible. I can try to look at it once more if we can avoid this as well.